src/node/nodestate.h

```diff
@@ -1067,6 +1067,7 @@ namespace ccf
       active_service->status = ServiceStatus::OPEN;
       service_view->put(0, active_service.value());
 
+      LOG_INFO_FMT("Changing network status from OPENING to OPEN");
```

Comment:
Should this be in the global hook for the service table instead? So that this is displayed on all nodes.

Reply:
And it's displayed when it's globally committed rather than tentatively executed on the leader? Yes!

Reply:
Yes. There's already a log message there but it should be more made explicit.

Reply:
Which it turns we were already doing, albeit not with the same wording.